app/scripts/modules/core/src/presentation/formsy/BasicFieldLayout.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from 'react';
+import { IFormFieldLayoutProps } from './formFieldLayout';
+
+/**
+ * A Form Field Layout component for Formsy Form Field components
+ *
+ * Accepts four react elements as props (_label, _input, _help, _error) and lays them out using bootstrap grid.
+ *
+ * +----------------div.form-group----------------------------+
+ * |                 +-------------div.col-md-9--------------+|
+ * |<label.col-md-3> |+-------------------------------------+||
+ * |                 ||input element                        |||
+ * |                 |+-------------------------------------+||
+ * |                 |                                       ||
+ * |                 |<help element>                         ||
+ * |                 |                                       ||
+ * |                 |<validation element>                   ||
+ * |                 |                                       ||
+ * |                 +---------------------------------------+|
+ * +----------------------------------------------------------+
+ */
+export class BasicFieldLayout extends React.Component<IFormFieldLayoutProps, {}> {
```

Comment:
This is the form field layout class that implements the markup/layout extracted from "Edit Entity Tag"